src/ucs/debug/instrument.h

```diff
@@ -117,5 +117,16 @@ void __ucs_instrument_record(uint64_t location, uint64_t lparam, uint32_t wparam
 
 #endif
 
+#if ENABLE_IB_TX_INSTRUMENTATION
+#define UCS_INSTRUMENT_IB_TX_RECORD UCS_INSTRUMENT_RECORD
```

Comment:
I did this to have different instrumentations types on/off without using if()-s in the code. Would you prefer using if()-s?